File: modul 6/PRAK601_2410817120001_Putri_Fatima_Azhara.py
baris, kolom = map(int, input().split())
isi = list (map(int, input().split()))
angka = []
x = 0

for i in range (baris):
    angka.append(isi[x:x+kolom])
    x += kolom
for baris in angka:
    for isi in baris:
        print(isi, end=' ')
    print()

File: modul 6/PRAK601_2410817120001_Putri_Fatima_Azhara.c
#include <stdio.h>

int main(){
    int baris, kolom, i, j, isi[20][20];
    scanf("%d %d", &baris, &kolom);

    for (i=0; i<baris; i++) {
        for (j=0; j<kolom; j++ ){
            scanf("%d", &isi[i][j]);
        }
    }

    for (i=0; i<baris; i++){
        for (j=0; j<kolom; j++){
            printf("%d ", isi[i][j]);
        }
        printf("\n");
    }
    return 0;
}